### ISSUE TITLE
>application/controllers/admin/dashboard.php [116]:

### ISSUE THREAD

**Issue by Kekely** [ushahidi/Ushahidi_Web#1437]
Non-static method protochart::chart() should not be called statically, assuming $this from incompatible context

**Comment by Kekely** [comment]
Non-static method members_Core::gravatar() should not be called statically, assuming $this from incompatible context

**Comment by poluster21** [comment]
Hi, I just installed the version 2.7.4 and throws me the same error.
You managed to fix it?
Do you have any leads?
Thank you

**Comment by polo49** [comment]
I am very appologize for my poor english. i am speaking french. i resolve this problem by modification of gravatar method in application/helpers/members.php. You add static in header of function "public static function gravatar($email, $s = 80, $d = 'mm', $r = 'g', $img = FALSE, $atts = array())". 
indeed many ushahidi method been deprecated.

**Comment by polo49** [comment]
Then when you have a same problem. You want to find a method in file situated in application/helper. thank

**Comment by bokac1984** [comment]
If anyone finds this, just as polo49 said, update all problematic class methods with keyword "static".
